commit a8df2ce5f69aa65c88b63ee7836d941ad7caaf61 (refs/changes/60/4560/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-07-24T10:42:21-07:00 (1 year, 2 months ago)
    
    MANTA-3849 'manta-adm update' takes a '--config-file' opt but does not use it
    Reviewed by: Kody A Kantor <kody@kkantor.com>
    Approved by: Kody A Kantor <kody@kkantor.com>

diff --git a/cmd/manta-adm.js b/cmd/manta-adm.js
index f277622..5b004aa 100755
--- a/cmd/manta-adm.js
+++ b/cmd/manta-adm.js
@@ -7,7 +7,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -538,13 +538,15 @@ MantaAdm.prototype.do_update = function (subcmd, opts, args, callback)
 };
 
 MantaAdm.prototype.do_update.help =
-    'Update deployment to match a JSON configuration.\n\n{{options}}';
+    'Update deployment to match a JSON configuration.\n\n' +
+    'Usage:\n\n' +
+    '    manta-adm update [OPTIONS] CONFIG-FILE [SERVICE]\n\n' +
+    '{{options}}';
 
 MantaAdm.prototype.do_update.options = [
     maCommonOptions.logFile,
     maCommonOptions.dryrun,
     maCommonOptions.confirm,
-    maCommonOptions.configFile,
 {
     'names': [ 'no-reprovision' ],
     'type': 'bool',
